lightning-invoice/src/lib.rs

```diff
@@ -97,29 +134,30 @@ fn __system_time_size_check() {
 /// # Panics
 /// If the check fails this function panics. By calling this function on startup you ensure that
 /// this wont happen at an arbitrary later point in time.
+#[cfg(feature = "std")]
```

Comment:
Maybe leave the function there on `no-std` but just make it always succeed?

Reply:
done